enhancements/baremetal-IPI/baremetal.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Neat-Enhancement-Idea
+authors:
+  - "@sadasu"
+reviewers:
+  - TBD
+  - "@abhinavdahiya"
+approvers:
+  - TBD
+  - "@abhinavdahiya"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: implemented
+---
+
+# Adding Baremetal IPI capabilities to OpenShift
+
+This enhancement serves to provide the context for a whole slew of features
+and enhancements that will follow to make Baremetal IPI deployments via
+OpenShift a reality.
+
+At the time of this writing, code for some of the enhancements have already
+merged, some are in progress and others are yet to me implemented. References
+to all these features in different stages of development will be provided
+below.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+Baremetal IPI deployments enable OpenShift to enroll baremetal servers to become
+Machines that can be consumed to run K8s workloads.
+The Baremetal Operator (https://github.com/metal3-io/baremetal-operator) along
+with other provisioning services (Ironic and dependencies) run in their own
+pod called the "metal3". This pod is deployed by the Machine API Operator when
+the Platform type is Baremetal. The OpenShift Installer is responsble for
+providing all the necessary configs required for a successful deployment.
+
+## Motivation
+
+The motivation for this enhancement request is to provide a background for all the
+the subsequent enhancement requests for Baremetal IPI deployments.
+
+### Goals
+
+The goal of this enhancement request is to provide context for all the changes
+that have already been merged towards making Baremetal IPI deployments a reality.
+All future Baremetal enhancement requests will refer back to this one to provide
+context.
+
+### Non-Goals
+
+No development PRs will be raised as a result of this enhancement request.
```

Comment:
This may be a double negative. The non-goal (IE: what will NOT be done) is no development prs [....].

Reply:
Fixed it.